ts/session/messages/outgoing/controlMessage/group/ClosedGroupMessage.ts

```diff
@@ -1,17 +1,18 @@
 import { SignalService } from '../../../../../protobuf';
 import { PubKey } from '../../../../types';
-import { ExpirableMessage, ExpirableMessageParams } from '../../ExpirableMessage';
+import { DataMessage } from '../../DataMessage';
+import { ExpirableMessageParams } from '../../ExpirableMessage';
 
 export interface ClosedGroupMessageParams extends ExpirableMessageParams {
   groupId: string | PubKey;
 }
 
-export abstract class ClosedGroupMessage extends ExpirableMessage {
+export abstract class ClosedGroupMessage extends DataMessage {
```

Comment:
Damn it's been a while with this code. Is this correct?

Reply:
I forgot why I did that, but also we are deprecating legacy group ,right?

Reply:
true. Fine for now

Reply:
Resolving